### DISCUSSION TITLE
GitHub sponsors

### DISCUSSION THREAD

**Discussion by CodeCampEdit** [community/community#15409] [Other Feature Feedback, Questions, & Ideas]
applied for github sponsors a week ago and has not been approved, what might be the problem?

**Comment by MrKrishnaAgarwal** [comment]
is it approved.
if it's not
you might have filled details incorrect
try filling again

**Comment by melvincwng** [comment]
Hi @CodeCampEdit, you may need to contact Github Support to follow up with this issue.

Please follow the guide here for instructions: https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket

Hope this helps to resolve your problem.

**Comment by yu199195** [comment]
applied for github sponsors a week ago and has not been approved, How long will it take?

**Reply by melvincwng** [reply]
@yu199195, I have addressed your question [here](https://github.com/github-community/community/discussions/19014)
